vmdb/app/models/miq_ae_yaml_export.rb

```diff
@@ -23,7 +23,7 @@ def write_model
     else
       @domain == ALL_DOMAINS ? write_all_domains : write_domain
     end
-    write_manifest(swap_domain_name(@domain))
+    write_manifest(@options['export_as'] || @domain)
```

Comment:
You should use `.presence` here: `write_manifest(@options['export_as'].presence || @domain)`